src/main/scala/chisel3/simulator/package.scala

```diff
@@ -127,30 +127,35 @@ package object simulator {
       // been moved.
       val movedFiles = mutable.HashSet.empty[Path]
       val supportArtifactsPath = Paths.get(workspace.supportArtifactsPath)
+      def moveFile(file: Path): Unit = {
+        // Convert the files to an absolute version and a relative version.
+        val (_abs, rel) = file match {
+          case file if file.startsWith(supportArtifactsPath) =>
+            (file, file.subpath(supportArtifactsPath.getNameCount(), file.getNameCount()))
+          case file => (supportArtifactsPath.resolve(file), file)
+        }
+        // Normalize the absolute path so it can be checked if it has already
+        // been moved.
+        val abs = _abs.normalize()
+        // Move the file into primarySourcesPath if it has not already been moved.
+        (abs, rel) match {
+          case (abs, _) if movedFiles.contains(abs) =>
+          case (abs, rel) =>
+            val dest = Paths.get(workspace.primarySourcesPath).resolve(rel)
+            dest.getParent.toFile.mkdirs
+            Files.move(
+              abs,
+              dest
+            )
```

Comment:
This is the nittiest of nits but I would make this 1 line, your changes seem to make it no longer need to be split.

Reply:
I noticed this and was relying on formatting the handle it, though it didn't fix it. I was very confused. Apparently the formatting won't touch this.

This is fixed here: eac5decc5